### ISSUE TITLE
Can someone please help!

### ISSUE THREAD

**Issue by gauthamvr** [InstaPy/InstaPy#5757]
I have been getting the same error whatever I do. I guess the error is with the session creation and login. Can someone please tell me the solution?.

Workspace in use: "C:/Users/Overgoat/InstaPy"
Traceback (most recent call last):
  File "C:/Users/Overgoat/PycharmProjects/instafollow5/main.py", line 6, in <module>
    session = InstaPy(username='removed them to paste here', password='removed them to paste here', headless_browser=False)
  File "C:\Users\Overgoat\PycharmProjects\instafollow5\venv\lib\site-packages\instapy\instapy.py", line 312, in __init__
    self.browser, err_msg = set_selenium_local_session(
  File "C:\Users\Overgoat\PycharmProjects\instafollow5\venv\lib\site-packages\instapy\browser.py", line 121, in set_selenium_local_session
    browser = webdriver.Firefox(
  File "C:\Users\Overgoat\PycharmProjects\instafollow5\venv\lib\site-packages\selenium\webdriver\firefox\webdriver.py", line 170, in __init__
    RemoteWebDriver.__init__(
  File "C:\Users\Overgoat\PycharmProjects\instafollow5\venv\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 157, in __init__
    self.start_session(capabilities, browser_profile)
  File "C:\Users\Overgoat\PycharmProjects\instafollow5\venv\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 252, in start_session
    response = self.execute(Command.NEW_SESSION, parameters)
  File "C:\Users\Overgoat\PycharmProjects\instafollow5\venv\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\Overgoat\PycharmProjects\instafollow5\venv\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.SessionNotCreatedException: Message: Unable to find a matching set of capabilities

**Comment by joebyjo** [comment]
same prob with me

**Comment by LeydenJar** [comment]
This is normally related with selenium not finding the webdriver. I would try deleting geckodriver, updating firefox and then letting instapy download the driver again next time it runs

**Comment by gauthamvr** [comment]
Where is the geckodriver downloaded by Instapy located? I have deleted the geckodriver which I have manually downloaded, then re-installed firefox, and ran the program again. Still the same error. It would be really helpful if someone could guide me.

**Comment by LeydenJar** [comment]
@gauthamvr 

geckodriver should be inside this folder:
C:/Users/Overgoat/InstaPy/assets/

Can you post your code and tell us what you did in order to run the program?
Are you on windows? was it working before?

**Comment by gauthamvr** [comment]
I have deleted the entire gecko drivers present in the assets folder(including the zip file), tried again. It downloaded new drivers, but the results were the same. I am getting the same error. Then I manually downloaded the gecko driver and replaced the default and tried running the same. Still the same error.
Yes, I'm running on windows. 
I have found a temporary solution by finding the piece of code in instapy, which searches for the gecko driver in the system. Then I have replaced the path with the location(path) of geckodriver that I have manually downloaded.

This is what I was trying to run,

from instapy import InstaPy
from random import randint

session = InstaPy(username="removed them to paste here", password="removed them to paste here")
session.login()
session.set_quota_supervisor(enabled=True, peak_comments_daily=20, peak_comments_hourly=21,peak_follows_daily=100,
                             peak_follows_hourly=10)
session.set_relationship_bounds(enabled=True, max_followers=1000, min_following=10)
session.set_action_delays(enabled=True,like=randint(3,7),comment=randint(3,7),follow=randint(3,7))
session.like_by_tags(["kerala"], amount=5)
session.set_dont_like(["naked", "nsfw"])
session.set_do_follow(True, percentage=100)
session.set_do_comment(True, percentage=50)
session.set_comments(["Nice!", "Sweet!", "Beautiful :heart_eyes:"])

session.end()


The sad news is that I got banned for running this code just once. Any code alteration suggestions to avoid a ban would be really helpful. Else its pointless to use.
